settings.gradle

```diff
@@ -1,2 +1,2 @@
 include "wherehows-common", "hadoop-dataset-extractor-standalone", "metadata-etl", "data-model", "backend-service",
-  "web"
+  "web", "RestLiClient"
```

Comment:
Thanks for taking all the time reviewing this check in. Just renamed the module to restli-client and its related references/gradles. Please double check and let me know if any comment!